### PR TITLE
update: quinoa version 1.2.4 to 1.2.5

### PR DESCRIPTION
**Pull request by shivam-sharma7**
update: quinoa version 1.2.4 to 1.2.5

### CODE REVIEW

**Comment by shivam-sharma7** [comment]
review/ @jesperpedersen

**Comment by jesperpedersen** [comment]
Whitespace...

**Comment by shivam-sharma7** [comment]
> Whitespace...

I did't understand

**Comment by jesperpedersen** [comment]
You reindented the entire file. Keep the current whitespace spacing, just add the quinoa upgrade and yourself to `<developers>`

**Comment by shivam-sharma7** [comment]
> You reindented the entire file. Keep the current whitespace spacing, just add the quinoa upgrade and yourself to `<developers>`

no i just upgrade only quinoa version and add myself as developer but it's automaticaly reflecting as change in whole file.

**Comment by jesperpedersen** [comment]
Remember to always squash and rebase pull requests.

You can look at https://github.com/Hyperfoil/Horreum/pull/412/files to see your current changes. The changes can only reflect the issue raised.

**Comment by jesperpedersen** [comment]
It is DOS line feeds - you need to use Unix ones

**Comment by shivam-sharma7** [comment]
> Remember to always squash and rebase pull requests.
> 
> You can look at https://github.com/Hyperfoil/Horreum/pull/412/files to see your current changes. The changes can only reflect the issue raised.

I know as reviewer you have to face some problem with these change but I have't touch anything

**Comment by shivam-sharma7** [comment]
take your time to review it.

**Comment by jesperpedersen** [comment]
You need to run a tool like `dos2unix` on it

**Comment by shivam-sharma7** [comment]
can we merged it for now?

**Comment by shivam-sharma7** [comment]
> You need to run a tool like `dos2unix` on it

thanks

**Comment by jesperpedersen** [comment]
Pull requests are merged based on minimal changes for the improvement in question. And, squashed and rebased commits is needed.

One improvement - one commit - which is up-to-date

**Comment by shivam-sharma7** [comment]
> One improvement - one commit - which is up-to-date

you mean i need to push only one commit instead it' having currently two commits

**Comment by jesperpedersen** [comment]
Yes, only one commit.

https://www.git-tower.com/learn/git/faq/git-squash - and force push aka `git push -f origin update-quinoa`

**Comment by shivam-sharma7** [comment]
> Yes, only one commit.
> 
> https://www.git-tower.com/learn/git/faq/git-squash - and force push aka `git push -f origin update-quinoa`

Thanks Done ✅

File: pom.xml [hunk 1]
```diff
@@ -126,6 +126,12 @@
             <organization>Red Hat</organization>
             <organizationUrl>http://www.redhat.com</organizationUrl>
         </developer>
+        <developer>
+            <name>Shivam Sharma</name>
+            <email>meshivam81@gmail.com</email>
+            <organization>Red Hat</organization>
```
Comment (jesperpedersen) [inline pom.xml:132]: Remove

File: pom.xml [hunk 2]
```diff
@@ -126,6 +126,12 @@
...
+            <organizationUrl>http://www.redhat.com</organizationUrl>
```
Comment (jesperpedersen) [inline pom.xml:133]: Remove

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (shivam-sharma7) [reply pom.xml]: Removed

**Comment by jesperpedersen** [comment]
... We are back with DOS line feeds ...

**Comment by shivam-sharma7** [comment]
> ... We are back with DOS line feeds ...

Done ✅

**Comment by jesperpedersen** [comment]
Rebased -- please, keep this in mind, and merged.

Thanks for your contribution !

**Comment by shivam-sharma7** [comment]
> Rebased -- please, keep this in mind, and merged.
> 
> Thanks for your contribution !

Thanks

**Comment by shivam-sharma7** [comment]
> and yourself to `<developers>`

mean?

**Comment by jesperpedersen** [comment]
You can raise another pull request with

```
  <developer>
      <name>...</name>
      <email>...</email>
  </developer>
```

in `pom.xml`.

**Pull request merged**